algorithms/gr-dij/python3/dijkstra.py

```diff
@@ -0,0 +1,83 @@
+# Library for INT_MAX
+import sys
+
+
+class Dijkstra:
+
+    def __init__(self, vertices):
+        self.V = vertices
+        self.graph = [[0 for column in range(vertices)]
+                      for row in range(vertices)]
+
+    def print_solution(self, dist):
+        print("Vertex Distance from Source")
+        for node in range(self.V):
+            print(node, "tt", dist[node])
+
+    # A utility function to find the vertex with
+    # minimum distance value, from the set of vertices
+    # not yet included in shortest path tree
+    def minDistance(self, dist, split_set):
+
+        # Initialize minimum distance for next node
+        min = sys.maxsize
```

Comment:
Codacy found an issue: [Redefining built-in 'min'](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6353492)

Reply:
i will remove this